components/camel-qdrant/src/test/java/org/apache/camel/component/qdrant/QdrantUpsertManualIT.java

```diff
@@ -31,7 +31,7 @@
 import static io.qdrant.client.ValueFactory.value;
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class QdrantUpsertTest extends QdrantTestSupport {
+public class QdrantUpsertManualIT extends QdrantTestSupport {
```

Comment:
These are other changes in components (`camel-qdrant` and `camel-lucene`) are outside the context of this PR.

Reply:
1. Why would these tests break anything here? 
2. These tests are not manual tests 

So, leave them outside the scope of this change.

Reply:
My apologies for renaming the Quadrant test classes in error. When I started working on these fixes, the Quadrant tests were failing due to Docker errors, so I assumed that they needed to be run manually. The tests are no longer failing, so I have reverted the class names. Thanks for catching this.